hoot-core-test/src/test/cpp/hoot/core/ops/MapCropperTest.cpp

```diff
@@ -257,7 +257,7 @@ class MapCropperTest : public HootTestFixture
     //compare relations
     const RelationMap relations = map->getRelations();
     HOOT_STR_EQUALS(1, relations.size());
-    QString relationStr = "relation(-1592); type: multipolygon; members:   Entry: role: outer, eid: Way:-1556;   Entry: role: inner, eid: Way:-1552; ; tags: landuse = farmland; status: invalid; version: 0; visible: 1; circular error: 15";
+    QString relationStr = "relation(-1592); type: multipolygon; members:   Entry: role: outer, eid: Way:-1556;   Entry: role: inner, eid: Way:-1552; ; tags: landuse = farmland; status: invalid; version: 0; visible: 1";
```

Comment:
Because of these changes I was able to remove circular error tags from a few different files where they weren't initially and our code added them.